go-controller/pkg/ovn/service.go

```diff
@@ -165,6 +167,130 @@ func (ovn *Controller) syncServices(services []interface{}) {
 	}
 }
 
+func (ovn *Controller) createService(service *kapi.Service) error {
+	klog.V(5).Infof("Creating service %s", service.Name)
+	if !util.IsClusterIPSet(service) {
+		return fmt.Errorf("no cluster IP for service %s found", service.Name)
+	} else if len(service.Spec.Ports) == 0 {
+		return fmt.Errorf("no Ports specified for service")
+	}
```

Comment:
Not having ports is not an issue, it will be the case for Statefulsets services. So, perhaps you should log debug message and `return nil`

Reply:
will fix